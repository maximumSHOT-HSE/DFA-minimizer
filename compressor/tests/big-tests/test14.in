15 90 6
1 3 0
1 10 1
1 14 2
1 1 3
1 7 4
1 12 5
2 15 0
2 14 1
2 14 2
2 3 3
2 4 4
2 3 5
3 9 0
3 8 1
3 13 2
3 3 3
3 13 4
3 9 5
4 7 0
4 7 1
4 2 2
4 5 3
4 11 4
4 13 5
5 15 0
5 14 1
5 15 2
5 12 3
5 13 4
5 6 5
6 11 0
6 13 1
6 2 2
6 2 3
6 9 4
6 4 5
7 15 0
7 7 1
7 12 2
7 9 3
7 14 4
7 14 5
8 13 0
8 15 1
8 2 2
8 3 3
8 6 4
8 4 5
9 3 0
9 6 1
9 7 2
9 2 3
9 7 4
9 5 5
10 8 0
10 2 1
10 6 2
10 2 3
10 9 4
10 11 5
11 6 0
11 12 1
11 4 2
11 14 3
11 2 4
11 8 5
12 9 0
12 12 1
12 2 2
12 3 3
12 12 4
12 1 5
13 14 0
13 5 1
13 8 2
13 5 3
13 15 4
13 12 5
14 4 0
14 11 1
14 6 2
14 1 3
14 15 4
14 12 5
15 10 0
15 11 1
15 1 2
15 4 3
15 5 4
15 8 5
10
7
7 3 6 5 14 9 7 
