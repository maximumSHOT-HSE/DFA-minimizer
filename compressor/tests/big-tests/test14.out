15 90 6
15 1 2
1 1 3
14 1 3
12 1 5
10 2 1
4 2 2
6 2 2
8 2 2
12 2 2
6 2 3
9 2 3
10 2 3
11 2 4
1 3 0
9 3 0
2 3 3
3 3 3
8 3 3
12 3 3
2 3 5
14 4 0
11 4 2
15 4 3
2 4 4
6 4 5
8 4 5
13 5 1
4 5 3
13 5 3
15 5 4
9 5 5
11 6 0
9 6 1
10 6 2
14 6 2
8 6 4
5 6 5
4 7 0
4 7 1
7 7 1
9 7 2
1 7 4
9 7 4
10 8 0
3 8 1
13 8 2
11 8 5
15 8 5
3 9 0
12 9 0
7 9 3
6 9 4
10 9 4
3 9 5
15 10 0
1 10 1
6 11 0
14 11 1
15 11 1
4 11 4
10 11 5
11 12 1
12 12 1
7 12 2
5 12 3
12 12 4
1 12 5
13 12 5
14 12 5
8 13 0
6 13 1
3 13 2
3 13 4
5 13 4
4 13 5
13 14 0
2 14 1
5 14 1
1 14 2
2 14 2
11 14 3
7 14 4
7 14 5
2 15 0
5 15 0
7 15 0
8 15 1
5 15 2
13 15 4
14 15 4
10
6
3 5 6 7 9 14 
